sql/sql_test.ts

```diff
@@ -0,0 +1,44 @@
+import { asserts } from "../_common/deps.ts";
```

Comment:
Is this file solely for sqlite tests? if so we can maybe rename this file to `sqlite_test.ts`

Reply:
TBD, but the *intent* is for the file to test `sql.ts`, but for now I'm using the sqlite driver to do so. Later I might change it to use a mock or to use multiple test drivers.